aws/resource_aws_launch_template_test.go

```diff
@@ -1658,3 +1753,30 @@ resource "aws_launch_template" "test" {
 }
 `, rName, enabled)
 }
+
+func testAccAWSLaunchTemplateConfig_defaultVersion(rInt int, version int) string {
+	return fmt.Sprintf(`
+resource "aws_launch_template" "test" {
+  name = "test_%d"
+  default_version = %d
+  user_data = "bar"
```

Comment:
I would suggest removing this since its not in `testAccAWSLaunchTemplateConfig_basic` so we can verify that solely changing the default version does not create a new version. 👍 

```suggestion
```

Reply:
since `name` is a force_new attribute, I updated to ensure we don't change that field but instead use another a schema attribute to trigger an "Update"..the last step in `TestAccAWSLaunchTemplate_defaultVersion`, however, ensures the condition where solely changing the default version does not create a new version